documentation/getting-started.md

```diff
@@ -1,64 +1,29 @@
 # Get started
```

Comment:
As this is the first documentation page you will go to when interested in the Barista components, some "welcoming introduction text" would be nice here.

E.g. something like: "On this page you find all steps you need to install the Barista components in your Angular project. Check out the components documentation for detailed information about the usage of single components." or similar...